<launch>

	<node pkg="asl_turtlebot" type="navigator.py" name="navigator" output="screen"/>

	<node pkg="asl_turtlebot" type="goal_supervisor.py" name="goal_supervisor" output="screen"/>

	<!-->
	<node pkg="asl_turtlebot" type="laser_pointer_filter.py" name="laser_pointer_filter" output="screen"/>
	<-->

	<node pkg="asl_turtlebot" type="vending_planner.py" name="vending_planner" output="screen"/>

	<node pkg="asl_turtlebot" type="food_localizer.py" name="food_localizer" output="screen"/>

	<node pkg="asl_turtlebot" type="display_food_location.py" name="display_food" output="screen"/>

	<node pkg="asl_turtlebot" type="display_goal_location.py" name="display_goal" output="screen"/>

	<!-->
	<node pkg="asl_turtlebot" type="request_publisher.py" name="request_publisher" output="screen"/>

	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find package_name)/rviz/config_file.rviz" />
	<-->
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(asl_turtlebot)/rviz/project_rviz.rviz" />


	 <node pkg="asl_turtlebot" type="goal_commander.py" name="goal_commander" output="screen"/>

</launch>
